api/inventorypb/agents.proto

```diff
@@ -293,7 +293,7 @@ message QANPostgreSQLPgStatementsAgent {
   AgentStatus status = 9;
 }
 
-// RDSExporter runs on Generic or Container Node and exposes RemoteRDS Node metrics.
+// RDSExporter runs on RemoteRDS Node and exposes RemoteRDS Node metrics.
```

Comment:
No, it really runs on Generic or Container node – we can't run processes on remote nodes.